_episodes/04-building-containers.md

```diff
@@ -144,6 +146,7 @@ We will automate this in the next section.
 > > ...
 > > ~~~
 > > {: .output}
-> > Notice how we did not need `--fakeroot` for the installation since pip installs user packages.
+> > Notice how we did not need neither `--writable ` nor `--fakeroot` for the installation, but everything worked fine since pip installs user packages, in the user $HOME directory.
```

Comment:
The whole sandbox use does. The idea is that you experiment w/ the sandbox so you try, do and undo as needed and then you use a description file and system locations. I meant to stress it on the video and there is a sentence at the beginning, it may be worth to add a reminder here

Reply:
changes coming...

Reply:
clarification added, ok to review and go ahead